net/nginx/Makefile

```diff
@@ -382,7 +378,7 @@ endef
 $(eval $(call module,lua, +luajit,ngx_http_lua, Enable Lua module))
 $(eval $(call module,stream, +@NGINX_STREAM_CORE_MODULE,ngx_stream, Add support for NGINX request streaming.))
 $(eval $(call module,ubus, +libubus +libjson-c +libblobmsg-json +@NGINX_UBUS,ngx_http_ubus, Enable UBUS api support directly from the server.))
-$(eval $(call module,dav-ext, +@NGINX_DAV,ngx_http_dav_ext, Enable the WebDAV methods PROPFIND OPTIONS LOCK UNLOCK.))
+$(eval $(call module,dav-ext, +@NGINX_DAV +libxml2,ngx_http_dav_ext, Enable the WebDAV methods PROPFIND OPTIONS LOCK UNLOCK.))
```

Comment:
Since dav-ext depends on `NGINX_DAV`, and `+NGINX_DAV:libxml2` exists for nginx-ssl, `+libxml2` exists for nginx-full, why is `+libxml2` here necessary?

Reply:
.so dependency scan complain on creating the separate package for dav-ext. the dependency is present for dav for nginx but is also required for the separate module dav-ext